WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java

```diff
@@ -167,6 +173,17 @@ protected void onPause() {
     @Override
     public boolean onOptionsItemSelected(final MenuItem item) {
         if (item.getItemId() == android.R.id.home) {
+            if (mIsFeed) {
```

Comment:
I'm not clear on the purpose of this addition, which suggests it needs to be commented.